api/v1alpha1/spinapp_types.go

```diff
@@ -99,6 +91,9 @@ type SpinAppStatus struct {
 
 	// Represents the current number of active replicas on the application deployment.
 	ReadyReplicas int32 `json:"readyReplicas"`
+
+	// Selector for the underlying pods
+	Selector string `json:"selector"`
```

Comment:
@endocrimes would we have to make this optional to support runtimes where this selector is meaningless? Or is setting it to the empty string sufficient.